nilearn/datasets.py

```diff
@@ -26,6 +25,17 @@
 
 import nibabel
 
+# Define _get_time, which is used module-wide to get the current time.
+try:
+    # Use uptime (when available) to avoid reporting time while computer slept.
+    import uptime
+    if uptime.uptime() is None:
+        raise Exception("uptime is not supported by this OS.")
+    _get_time = uptime.uptime
```

Comment:
I suggest to move on.  

If so, the first commit contains the original changes and none of this uptime stuff.  Perhaps it can be cherry-picked into master?  Or do you prefer I force the change to my branch, so we can merge here?